geowebcache/core/src/main/java/org/geowebcache/GeoWebCacheDispatcher.java

```diff
@@ -12,7 +12,10 @@
  *  You should have received a copy of the GNU Lesser General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  * 
- * @author Arne Kepp, The Open Planning Project, Copyright 2008
+ * Copyright OpenPlans 2008-2014
+ * Copyright OSGeo 2014
+ * 
+ * @author Arne Kepp, The Open Planning Project
```

Comment:
This look fine, I usually do:

```
 Copyright 2008-2014 Open Plans
 Copyright 2014 Open Source Geospatial Foundation
```

Or shorter

```
 (c) 2008-2014 Open Plans
 (c) 2014 Open Source Geospatial Foundation
```

But since you are doing the pull request you can make your own tradition.